Dockerfile

```diff
@@ -4,15 +4,15 @@ LABEL maintainer="team@semantic.works"
 
 RUN apt-get update && apt-get -y upgrade && apt-get -y install git openssh-client rsync
 
-ENV MU_SPARQL_ENDPOINT 'http://database:8890/sparql'
-ENV MU_APPLICATION_GRAPH 'http://mu.semte.ch/application'
-ENV NODE_ENV 'production'
+ENV MU_SPARQL_ENDPOINT='http://database:8890/sparql'
+ENV MU_APPLICATION_GRAPH='http://mu.semte.ch/application'
+ENV NODE_ENV='production'
 
-ENV HOST '0.0.0.0'
-ENV PORT '80'
+ENV HOST='0.0.0.0'
+ENV PORT='80'
 
-ENV LOG_SPARQL_ALL 'true'
-ENV DEBUG_AUTH_HEADERS 'true'
+ENV LOG_SPARQL_ALL='true'
+ENV DEBUG_AUTH_HEADERS='true'
 
 WORKDIR /usr/src/app
 COPY package.json /usr/src/app/package.json
```

Comment:
no need to add a package lock anymore. The service can choose to include/commit one now as the merged package-lock is generated by the mu install script